### PR TITLE
Feature/#79 updated privacy page

### LINKED ISSUE #79: Change privacy page

### PR DESCRIPTION
**Pull request by altarin**
closes #79

### CODE REVIEW

File: makeup/html/conditions.html [hunk 1]
```diff
@@ -5,154 +5,142 @@
...
+        <p>1.2. Для осуществления регистрации на кэшбэк-сервисе пользователь должен ознакомиться с настоящими
+            условиями, и принять их. Условия могут периодически меняться без уведомления пользователей, Изменения
+            условий вступают в силу после их публикации на сайте.</p>
 
-        </div>
-        <div class="col-sm-3"></div>
+        <h5>2. Термины и определения</h5>
+        <p></p>
+        <p><i>Кэшбэк</i> - Возврат части стоимости покупки в интернет магазинах в виде денежных средств.</p>
+        <p><i>CashTan.tk</i> - сервис, который возвращает часть денег с покупок, сделанных в интернет магазинах,
```
Comment (IvanAlekseevichPopov) [inline makeup/html/conditions.html]: site address must be in lower case.

File: makeup/html/conditions.html [hunk 2]
```diff
@@ -5,154 +5,142 @@
...
+        <p></p>
+        <h5> 4. Регистрация пользователя</h5>
+        <p></p>
+        <p>4.1. Пользователями кэшбэк-сервиса могут быть физические лица, достигшие возраста 18 лет. Регистрация на
+            сайте необходима для обеспечения достоверной и точной информации о пользователе.</p>
+        <p>4.2. Для регистрации на сайте пользователю необходимо ввести данные логина (электронной почты) и пароля в
+            регистрационной форме. Для исключения недоразумений просьба указывать действующий адрес электронной
+            почты.</p>
+        <p>4.3. После окончания процесса регистрации пользователю становится доступен Личный кабинет на сайте
+            CashTan.tk и на него распространяются все настоящие Условия со всеми вносимыми дополнениями и
```
Comment (IvanAlekseevichPopov) [inline makeup/html/conditions.html]: site address must be in lower case.

File: makeup/html/conditions.html [hunk 3]
```diff
@@ -5,154 +5,142 @@
...
+            изменениями.</p>
+        <p>4.4. Один пользователь может иметь только одну учетную запись.</p>
+        <p>4.5. Принимая данные Условия, пользователь соглашается, что кэшбэк-сервис не несет ответственности за
+            некорректные данные, указанные пользователем при регистрации.</p>
+        <p>4.6. Администрация кэшбэк-сервиса оставляет за собой право удалить учетную запись пользователя, в случае
+            если со стороны пользователя будут выявлены действия мошеннического и/или обманного характера.</p>
+        <p></p>
+        <h5> 5. Конфиденциальность</h5>
+        <p></p>
+        <p>5.1. Пользуясь сайтом CashTan.tk, пользователь предоставляет информацию об электронной почте, пароле,
```
Comment (IvanAlekseevichPopov) [inline makeup/html/conditions.html]: site address must be in lower case.

File: makeup/html/conditions.html [hunk 4]
```diff
@@ -5,154 +5,142 @@
...
+            своих контактных данных.</p>
+        <p>5.2. Сбор данной информации проводится с целью оказания пользователям оговоренного сервиса с лучшим
+            качеством и в минимальные сроки.</p>
+        <p>5.3. Администрация кэшбэк-сервиса CashTan.tk принимает все возможные меры по обеспечению
```
Comment (IvanAlekseevichPopov) [inline makeup/html/conditions.html]: site address must be in lower case.

File: makeup/html/conditions.html (hunk 1 above)
Comment (altarin) [reply makeup/html/conditions.html]: fixed

File: makeup/html/conditions.html (hunk 3 above)
Comment (altarin) [reply makeup/html/conditions.html]: fixed

File: makeup/html/conditions.html (hunk 4 above)
Comment (altarin) [reply makeup/html/conditions.html]: fixed

File: makeup/html/conditions.html (hunk 2 above)
Comment (altarin) [reply makeup/html/conditions.html]: fixed

**Review by IvanAlekseevichPopov — APPROVED** [review]

**Pull request merged**